package.json

```diff
@@ -3,40 +3,42 @@
   "version": "2.0.1",
   "license": "Apache-2.0",
   "description": "A class designed to enable easily extending the built-in javascript Error, allowing typed errors.",
-  "main": "src/typed-error.js",
-  "types": "src/typed-error.ts",
+  "main": "lib/typed-error.js",
```

Comment:
NIT: other projects use build/

Reply:
I'm not sure why I went with `lib/`, I thought I'd seen examples of it in other resin modules but I'm not sure where (aside `node-tunnel`, which I also wrote… but if I were deciding I would probably go with `dist/`).

Is there any sort of standard? Be it resin or typescript or community or whatever.

Reply:
I think typescript defaults to `build` if no folder is specified in `tsconfig.json`.

Reply:
It looks like if you don't specify an `outdir` then `tsc` will put the generated files alongside their source typescript.